docs/checkers/eslint.md

```diff
@@ -37,13 +37,35 @@ export default {
 }
 ```
 
+### Performance Optimization
+
+If you're experiencing `EMFILE: too many open files` errors or want to improve performance, you can use the `watchPath` option to limit file watching to specific directories:
+
+```js
+export default {
+  plugins: [
+    checker({
+      eslint: {
+        lintCommand: 'eslint "./src/**/*.{ts,tsx}"',
```

Comment:
This looks good, but I wonder if we could infer the watch path based on the `lintCommand`?